README.md

```diff
@@ -185,19 +258,18 @@ Note that **React-related error messages are typically significantly more helpfu
 ### Adding Additional Routes for the Dev Server
 As you add more routes to your front-end application, you will need to make the corresponding API for the dev server in `client/server.js`. See our example `server.js` from our [tutorial](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/server.js).
 
-## Additional Documentation:
-
-+ [Linters](docs/linters.md)
-+ [Manual Configuration](docs/manual_configuration.md)
-+ [Node Dependencies and NPM](docs/node_dependencies_and_npm.md)
+## Additional Reading
++ [Generated Client Code](docs/additional_reading/generated_client_code.md)
++ [Manual Installation](docs/additional_reading/manual_installation.md)
++ [Node Dependencies and NPM](docs/additional_reading/node_dependencies_and_npm)
 
 ## Contributing
-Bug reports and pull requests are welcome. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to our version of the [Contributor Covenant](contributor-covenant.org) code of conduct (see [CODE OF CONDUCT](CODE_OF_CONDUCT.md)).
+Bug reports and pull requests are welcome. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to our version of the [Contributor Covenant](contributor-covenant.org) code of conduct (see [CODE OF CONDUCT](docs/code_of_conduct.md)).
 
-See [Contributing](docs/Contributing.md) to get started.
+See [Contributing](docs/contributing.md) to get started.
 
 ## License
-The gem is available as open source under the terms of the [MIT License](LICENSE).
+The gem is available as open source under the terms of the [MIT License](docs/LICENSE).
 
 ## Authors
 [The Shaka Code team!](http://www.shakacode.com/about/)
```

Comment:
@justin808  We have a link to the forums several lines below this, added the specific category link to that